### ISSUE TITLE
Remember OPDS password

### ISSUE THREAD

**Issue by Mikescher** [geometer/FBReaderJ#381]
An option to remember the password for an network opds server would be great.

I have tested it with normal HTTP authentification. It prompts for the password, but doesn't remember it once you have totally quit the app.

**Comment by trefmanic** [comment]
Yes! This would be a nice addition. It's a hassle to enter long passwords on a mobile device.
